plasma_framework/docs/contracts/PaymentInFlightExitRouter.md

```diff
@@ -42,7 +42,7 @@ bool private bootDone;
 ```js
 event IFEBondUpdated(uint128  bondSize);
 event PiggybackBondUpdated(uint128  bondSize, uint128  exitBountySize);
-event InFlightExitStarted(address indexed initiator, bytes32 indexed txHash, bytes inFlightTx, uint256[] inputUtxosPos, bytes[] inFlightTxWitnesses bytes[] inputTxs);
+event InFlightExitStarted(address indexed initiator, bytes32 indexed txHash);
```

Comment:
why did this happen? didn't we actually add 
```
bytes inFlightTx, uint256[] inputUtxosPos, bytes[] inFlightTxWitnesses bytes[] inputTxs
```

Reply:
Just the generated docs coming up to date. The `InFlightExitStarted ` event in `PaymentStartInFlightExit.sol` is correct